packages/sources/layer2-sequencer-health/src/adapter.ts

```diff
@@ -34,7 +34,7 @@ export const makeExecute: ExecuteFactory<ExtendedConfig, endpoints.TInputParamet
   // Disclaimer on startup
   Object.keys(HEALTH_ENDPOINTS).forEach((network) => {
     if (!HEALTH_ENDPOINTS[network as Networks]?.endpoint)
-      Logger.info(`Health endpoint not available for network: ${network}`)
+      Logger.info(`[${network}] Health endpoint not available for network: ${network}`)
```

Comment:
we're always guaranteed to have a valid `network` here, right?

Reply:
Yep! This uses `HEALTH_ENDPOINTS`, which is configured in the EA so we know it has networks. 

For the other logs, we use the input param `network` which is a required param.